src/modules/getApi.js

```diff
@@ -0,0 +1,15 @@
+import leadList from '../index.js';
+
+async function getApi() {
+  const datar = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/p5yqiCio7d4FE7KGl6s2/scores';
+  const dataf = await fetch(datar);
+  const jsonFormat = await dataf.json();
+  leadList.innerHTML = '';
+  for (let i = 0; i < jsonFormat.result.length; i += 1) {
+    leadList.innerHTML
+            += ` <div class="score-board"> <p class="scores">
+     ${jsonFormat.result[i].user}: ${jsonFormat.result[i].score}</p> 
+     </div> `;
+  }
+}
```

Comment:
- [ ] Kindly change this function to arrow functions instead of the function keyword.

Reply:
well resolved! ✔️